src/routes/e2e/widget/inline/+page.svelte

```diff
@@ -28,9 +29,9 @@
   });
   journeyEvents.subscribe(
     (event: {
-      user: { successful: any; error: any };
-      journey: { error: any };
-      oauth: { error: any };
+      user: { successful: boolean; error: unknown };
+      journey: { error: unknown };
+      oauth: { error: unknown };
```

Comment:
I'd like to remove this type, if we can. It makes it inconsistent with the other e2e test page: https://github.com/ForgeRock/forgerock-web-login-framework/blob/beta/src/routes/e2e/widget/modal/%2Bpage.svelte#L31